### DISCUSSION TITLE
Cannot adjust my own volume with 'my' fader

### DISCUSSION THREAD

**Discussion by JacquesFreud** [jamulussoftware/jamulus#2952] [Technical Support & Help] [answered]
When playing with others one may want to set one's own volume lower or higher to match the other musician's volume levels.
When I want to adjust my volume either setting it to the max or to the minimum with the fader does not work. Not to my own ears nor to the ears of the other players. The sound level stays the same and I can only adjust it on my own mixer, before it goes out to Jamulus.. Then people start complaining I am not loud enough etc....

I recently updated mij M1 Mac to MacOS Ventura (13.0)

Anyone having the same problem? Then it may be a bug. Otherwise I am making mistakes (resets 'n all I have done, of course)

Thanks for thinking along,

Jacques

**Comment by hoffie** [comment]
> When I want to adjust my volume either setting it to the max or to the minimum with the fader does not work. Not to my own ears

Strange. That might imply that you aren't hearing yourself (solely) through Jamulus, but rather via some local monitoring function? I'd try to rule that out: E.g. do you hear yourself even when Jamulus is not started or not connected?

> nor to the ears of the other players.

That's expected. You are only controlling your own mix. The best way to adjust your real input level is to do that as close to the hardware as possible. I'd try using the Mac system settings here: https://support.apple.com/guide/mac-help/change-the-sound-input-settings-mchlp2567/mac

> I recently updated mij M1 Mac to MacOS Ventura (13.0)

I don't think we have any reports about Jamulus on that version (neither positive nor negative), so there might be changes, of course.

**Comment by JacquesFreud** [comment, marked as answer]
That was good thinking, Christian! I got confused about my settings on the mixer and one of my Jamulusfriends helped to solve the problem. Thanks!
Jacques
